srcs/parser/check_map_content.c

```diff
@@ -0,0 +1,45 @@
+
+#include <parser.h>
+
+static bool check_valid_char(char c)
+{
+	if (c != ' ' && c != '1' && c != '0'
+		&& c != 'N' && c != 'S' && c != 'W' && c != 'E')
+		return (false);
+	return (true);
+}
+
+bool	check_map_content(char **map_content)
+{
+	int x;
+	int x_size;
+	int y;
+	int y_size;
```

Comment:
same, this is unused, needs to be removed.

Reply:
inderdaad